SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -8848,7 +8877,10 @@ void CEulerSolver::BC_ActDisk(CGeometry *geometry, CSolver **solver_container, C
 //        /*--- Set the normal vector and the coordinates ---*/
 //
 //        visc_numerics->SetNormal(Normal);
-//        visc_numerics->SetCoord(geometry->nodes->GetCoord(iPoint), geometry->node[iPoint_Normal]->GetCoord());
```

Comment:
Apart from the obvious change I also replaced `geometry->node[iPoint_Normal]->GetCoord()` by `geometry->nodes->GetCoord(Point_Normal)`.